blazor/mention/working-with-data.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: Data Binding in Blazor Mention Component | Syncfusion
+description: Checkout and learn here all about Data Binding in Syncfusion Blazor Mention component and more.
+platform: Blazor
+control: Mention
+documentation: ug
+---
+
+# Working with Data in Mention
+
+The Mention loads the data either from the local data sources or remote data services. Using the `DataSource` property,  bind the local data or using the [DataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataManager.html), bind the remote data.
+
+* **TItem** - Specifies the type of the datasource of the Mention component.
+
+## Binding local data
+
+### Simple data
+
+The Mention has provided support to load an array of primitive data such as strings and numbers. Here, both the value and text fields act the same.
+
+{% highlight razor %}
+
+{% include_relative code-snippet/array-of-simple-data.razor %}
+
+{% endhighlight %}
+
+![Blazor Mention with array of simple data](./images/blazor-mention-data-binding.png)
+
+## JSON data
```

Comment:
Remove this section

Reply:
https://blazor.syncfusion.com/documentation/dropdown-list/data-binding#primitive-type

Refer this docs section